package.json

```diff
@@ -8,7 +8,8 @@
     "start": "npm run init && astro dev",
     "build": "astro check && astro build",
     "preview": "astro preview",
-    "astro": "astro"
+    "astro": "astro",
+    "postinstall": "npm install -g loam-cli"
```

Comment:
Instead of including `loam-cli` in the dependencies, I'm adding it as a postinstall script so that it can be installed globally. Otherwise it was being installed in the node_modules dir, and we'd need to run the binary like this: `./node_modules/loam-cli/bin/loam`

Reply:
Hmm in the package.json it should be listed as a  binary and then added to `./node_modules/.bin/loam` so that inside `package.json`, it will use that binary. Probably don't want to install it globally.

Reply:
That makes sense not to want to install globally. 


From the node docs:
> When this package is installed as a dependency in another package, the file will be linked where it will be available to that package either directly by npm exec or by name in other scripts when invoking them via npm run-script.

If I'm reading that section correctly, I think that if we add this to package.json:
```
  "bin": {
    "loam": "./node_modules/.bin/loam"
  }
```
we would still need to call the binary like this: `npm exec loam`.  Is that good enough? 

I also read one article that suggested running `npm link` after install... but that seemed to hang when including it in the `postinstall` script. Let me know if I'm missing another way to handle this! Happy to make an update.

Reply:
Update: I am able to include `npm link loam-cli` in the postinstall script... but I think this is still making a globally available thing that is a symlink. 🤔

Reply:
Looking at the `package.json` of `loam-cli`, it has
```
  "bin": {
    "loam": "run.js"
  },
  ```
  Would should then add `loam` to `node_modules/.bin`. I haven't tried it locally myself, but this is what has been the case for me in the past.